app/Http/Livewire/Cooperation/Frontend/Tool/SimpleScan/Form.php

```diff
@@ -182,13 +183,15 @@ public function save()
         $flowService->checkConditionals($dirtyToolQuestions, Hoomdossier::user());
 
         $quickScan = Scan::findByShort(Scan::QUICK);
-        $masterHasCompletedScan = $this->building->hasCompletedScan($this->scan, $this->masterInputSource);
+
+        $buildingService = BuildingService::init($this->building);
+
 
         // so this is another exception to the rule which needs some explaination..
         // we will only calculate the small measure when the user is currently on the lite scan and did not complete the quick-scan
         // this is done so when the user only uses the lite-scan the woonplan only gets small-measure, measureApplications.
         // else we will just do the regular recalculate/
-        if ($masterHasCompletedScan) {
+        if ($buildingService->cancalculate($this->scan)) {
```

Comment:
This works? Even with lowercase C?

Reply:
typo, thanks to php loosely typed goodness.